agents/PAW-02B Impl Planner.agent.md

```diff
@@ -279,25 +284,45 @@ After writing the plan to `.paw/work/<feature-slug>/ImplementationPlan.md`:
    **IF Review Strategy = 'prs'**:
    - Ensure on planning branch: `git branch --show-current`, create if needed: `git checkout -b <target_branch>_plan`
    - Stage artifacts: `git add .paw/work/<feature-slug>/{Spec.md,SpecResearch.md,CodeResearch.md,ImplementationPlan.md}` and prompt files
-   - Verify: `git diff --cached`, commit, push: `git push -u <remote> <target_branch>_plan`
+   - Verify: `git diff --cached`
+   - Commit following structured Commit Message Format:
+     - Extract Work Title and Work ID from WorkflowContext.md
+     - Format: `[<Work Title>] <type>: <description>` (omit Phase N for planning work)
+     - Use type `docs:` or `chore:` for planning artifacts
```

Comment:
`omit Phase N for planning work` is not needed here. Also, use "planning" as the type.